rollbar.js

```diff
@@ -89,13 +134,105 @@ exports.init = function (accessToken, options) {
     options = options || {};
     options.environment = options.environment || process.env.NODE_ENV || 'unspecified';
 
-    api.init(accessToken, options);
-    notifier.init(api, options);
-    initialized = true;
+    var api = this._api = new Api(accessToken, options);
+    var notifier = this._notifier = new Notifier(api, options);
+
+    Object.defineProperties(this, {
+      reportMessage: {
+        value: notifier.reportMessage.bind(notifier),
+        enumerable: true
+      },
+      reportMessageWithPayloadData: {
+        value: notifier.reportMessageWithPayloadData.bind(notifier),
+        enumerable: true
+      },
+      handleError: {
+        value: notifier.handleError.bind(notifier),
+        enumerable: true
+      },
+      handleErrorWithPayloadData: {
+        value: notifier.handleErrorWithPayloadData.bind(notifier),
+        enumerable: true
+      }
+    });
+
+    this._initialized = true;
   }
 };
 
 
+Rollbar.prototype.errorHandler = function (accessToken, options) {
+  /*
+   * A middleware handler for connect and express.js apps. For a list
+   * of supported options, see the init() docs above.
+   *
+   * All exceptions thrown from inside an express or connect get/post/etc... handler
+   * will be sent to rollbar when this middleware is installed.
+   */
+  this.init(accessToken, options);
+  return function (err, req, res, next) {
+    var cb = function (rollbarErr) {
+      if (rollbarErr) {
+        console.error('[Rollbar] Error reporting to rollbar, ignoring: ' + rollbarErr);
+      }
+      return next(err, req, res);
+    };
+
+    if (!err) {
+      return next(err, req, res);
+    }
+
+    if (err instanceof Error) {
+      return this.notifier.handleError(err, req, cb);
+    }
+
+    return this.notifier.reportMessage('Error: ' + err, 'error', req, cb);
+  };
+};
+
+
+Rollbar.prototype.handleUncaughtExceptions = function (accessToken, options) {
```

Comment:
It's a bit strange to have this be part of the Rollbar instance since we will never want to handle uncaught exceptions from more than one instance.